sdk/sdk-api/src/main/java/mil/emp3/api/MilStdSymbol.java

```diff
@@ -155,6 +155,63 @@ public static Affiliation fromString(String str) {
 
             return eValue;
         }
+
+        /**
+         * This static method returns the affiliation enumerated value for the given string representation.
+         * @param chr
+         * @return affiliation enumerated value or null if the string does not represent a valid MilStd affiliation code.
+         */
+        public static Affiliation fromChar(char chr) {
+            Affiliation eValue = null;
+
+            switch (chr) {
+                case 'P':
+                    eValue = PENDING;
+                    break;
+                case 'U':
+                    eValue = UNKNOWN;
+                    break;
+                case 'F':
+                    eValue = FRIEND;
+                    break;
+                case 'N':
+                    eValue = NEUTRAL;
+                    break;
+                case 'H':
+                    eValue = HOSTILE;
+                    break;
+                case 'A':
+                    eValue = ASSUMED_FRIEND;
+                    break;
+                case 'S':
+                    eValue = SUSPECT;
+                    break;
+                case 'G':
+                    eValue = EXERCISE_PENDING;
+                    break;
+                case 'W':
+                    eValue = EXERCISE_UNKNOWN;
+                    break;
+                case 'D':
+                    eValue = EXERCISE_FRIEND;
+                    break;
+                case 'L':
+                    eValue = EXERCISE_NEUTRAL;
+                    break;
+                case 'M':
+                    eValue = EXERCISE_ASSUMED_FRIEND;
+                    break;
+                case 'J':
+                    eValue = JOKER;
+                    break;
+                case 'K':
+                    eValue = FAKER;
+                    break;
```

Comment:
This will return `null` if the char doesnt match anything in the switch statement, is there a default value we can assume?

Reply:
Right, if we can't assign it an enum it should fail